Project/app.c

```diff
@@ -0,0 +1,81 @@
+
+
+#include "app.h"
+
+uint8_t gl_app_state = FALSE;
+
+int main(void)
+{
+	boolean val = TRUE;
+	(void)LED_init();
+	(void)BUTTON_init();
+
+	while(1)
+	{
+		(void)BUTTON_read(BUTTON_0,&val);
+		if(val == FALSE)
+		{
+			while(val == FALSE)
+			{
+				(void)BUTTON_read(BUTTON_0,&val);
+			}
+			
+			debouncing_handler();
+			gl_app_state++;
+		  
+
+		}
+    switch(gl_app_state)
```

Comment:
Switch case should have only one break

Reply:
updated successfully, you can check it now.